packages/cli/src/lib/merge/algorithms.js

```diff
@@ -0,0 +1,96 @@
+import util from 'util'
+
```

Comment:
I would strongly prefer if we didn't write all our own algorithms here, and just used lodash ones. The descriptions look complciated, but I'm sure there are standard functions to do these?

Reply:
I agree. Perhaps with the exception of `sieve`, which is a bit odd, I suspect there's good lodash equivalents for these.